sqlglot/expressions.py

```diff
@@ -266,7 +269,12 @@ def add_comments(self, comments: t.Optional[t.List[str]]) -> None:
         if self.comments is None:
             self.comments = []
         if comments:
-            self.comments.extend(comments)
+            for comment in comments:
+                _, *meta = comment.split(SQLGLOT_META)
```

Comment:
should we allow one to set >1 meta fields like `/* sqlglot.meta fld1 = foo, fld2 = bar ... */`?